ext/pdo_sqlite/tests/subclasses/pdo_sqlite_createcollation.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+PDO_sqlite: Testing sqliteCreateCollation()
+--EXTENSIONS--
+pdo_sqlite
+--FILE--
+<?php
+
+// This test was copied from the pdo_sqlite test for sqliteCreateCollation
+$db = new PdoSqlite('sqlite::memory:');
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+$db->query('CREATE TABLE IF NOT EXISTS test_pdo_sqlite_createcollation (id INT AUTO INCREMENT, name TEXT)');
```

Comment:
Does it need `IF NOT EXISTS`? Maybe forgot to fix this.

Reply:
Yes, I forgot it previously.